byte-streams-explainer.md

```diff
@@ -86,3 +102,24 @@ function makeReadableByteStream() {
 With this in hand, we can create and use BYOB readers for the returned `ReadableStream`. The adaptation between the
 low-level byte tracking of the underlying byte source shown here, and the higher-level chunk-based consumption of
 a default reader, is all taken care of automatically by the streams implementation.
+
+
+## Goals
+
+*   Support for {type: “bytes”} in the [ReadableStream constructor](https://streams.spec.whatwg.org/#rs-constructor).
```

Comment:
I think these are means rather than goals. How about the following?

 - Provide a way to represent a ReadableStream for bytes efficiently.
 - Avoid races caused by multiple access for the same buffer.

Reply:
Thank you for the suggestion! I've changed the goals section to have those points instead.